en/prepare/attributes-form.rst

```diff
@@ -114,6 +116,17 @@ The option "hyperlink" on the attachment widget will disable the camera function
 
  .. figure:: /images/hyperlink_option.png
 
+Value Relation
+--------------
+
+The widget "Value Relation" offers values from a related table in a combobox. You can select layer, key column and value column. Several options are available to change the standard behaviors: allow null value, order by value, allow multiple selections and use of auto-completer. The forms will display either a drop-down list or a line edit field when completer checkbox is enabled.
```

Comment:
```suggestion
The widget "Value Relation" offers values from a related table in a combobox. You can select layer, key column and value column. Several options are available to change the standard behaviors: allow null value, order by value and allow multiple selections. Auto-completer option is yet to be supported. The widget will display a drop-down or multiple-select list of the referenced features.
```
I am afraid the auto-completer is only on the todo list?

Reply:
In this case I would completely drop the statement about auto completion. Or move it to the end, more as a note.